app/page.tsx

```diff
@@ -20,15 +25,26 @@ export default async function Home() {
     return (
       <div className="px-2 my-6 space-y-2 w-full">
         <Header />
-        <ClientLayout />
-        <Suspense fallback={<Spinner />}>
-          <EventSection
-            streamedDataPast={pastDates}
-            streamedDataFuture={futureDates}
-            currentDate={currentDate}
-            today={today}
-          />
-        </Suspense>
+        <div className="relative w-full">
+          <div className="flex flex-col space-y-24">
+            <HeroHome />
+            <HeroCard />
+            <ImpactBanner />
+            <FeaturedCarousel />
+          </div>
+        </div>
+        <div id="events" className="mt-20 w-full">
+          <SectionHeader href={"#events"} title={"Events"} />
```

Comment:
I think now you can delete the `href={"#events"}` ?

Reply:
Ah seems like it's a required prop of SectionHeader. It's ok we'll be getting rid of that SectionHeader component in an upcoming change anyway!